### PR TITLE
find usages should be full screen

### PR DESCRIPTION
**Pull request by emord**
@czue bug you mentioned

image

That's pretty ugly, so making find usages full screen for now

buddy @esoergel

### CODE REVIEW

File: src/less-style/modals.less [hunk 1]
```diff
@@ -37,7 +37,7 @@
 }
 
 // Full-screen modal for displaying form XML
-.fd-source-modal {
+.fd-full-screen-modal {
```
Comment (orangejenny) [inline src/less-style/modals.less]: Doesn't this rename mean you can remove `fd-source-modal` from `core.js`? I don't see any other usages of `fd-source-modal`.

File: src/core.js [hunk 2]
```diff
@@ -725,6 +725,7 @@ define([
             form = _this.data.core.form,
             tableData = form.findUsages();
 
+        $modal.addClass('fd-find-usages-modal fd-full-screen-modal');
```
Comment (orangejenny) [inline src/core.js]: What's the purpose of the `fd-find-usages-modal` class?

**Comment by czue** [comment]
thanks J$

File: src/less-style/modals.less (hunk 1 above)
Comment (emord) [inline src/less-style/modals.less]: I removed those. I initially had them separately and hesitated to remove something descriptive

Comment (orangejenny) [inline src/less-style/modals.less]: sorry, forgot I was looking at individual commits

**Pull request merged**